package.json

```diff
@@ -7,10 +7,10 @@
     "email": "oleg008@gmail.com"
   },
   "peerDependencies": {
-    "babel-eslint": "^7.1.0",
-    "eslint": "^3.6.0",
-    "eslint-config-airbnb": "^13.0.0",
-    "eslint-plugin-react": "^6.3.0"
+    "babel-eslint": "*",
```

Comment:
why * instead of specific version?
why adding rules instead of using airbnb's default? Is it in order to stay consistent with the current code base?

Reply:
-  `*` because the versions were quite old, giving a lot of warnings with newer eslint plugins/configs. I'll put either recent `^x.x.x` or `>=x.x.x` it should work for any version actually
- I can try to remove some rules, and update more the code in react-jss to pass linting

 sometimes it's annoying

`  const {theming = defaultTheming, inject, jss: optionsJss, ...sheetOptions} = options` with https://github.com/airbnb/javascript/blob/master/packages/eslint-config-airbnb-base/rules/style.js#L384-L389 will need  to multiline,